.eslintignore

```diff
@@ -0,0 +1,9 @@
+**/node_modules/*
+**/vendor/*
+**/*.min.js
+/*.md
+/package.json
+/npm-debug.log
+/procfile
+/test/test_bundle.js
```

Comment:
Try replacing this line with ```**/test/test_bundle.js```  Currently it is still not being ignored.  You can see what is getting thrown as a linter error by typing eslint * in your terminal.